snap/snapcraft.yaml

```diff
@@ -29,7 +29,6 @@ parts:
     - git
     override-pull: |
         snapcraftctl pull
-        snapcraftctl set-version $(cd $SNAPCRAFT_PART_SRC && git describe --tags)
```

Comment:
For some reason there were no tags present when built by the action. Using a more appropriate `dvc --version` below instead.